### DISCUSSION TITLE
i3 on multiple tty's?

### DISCUSSION THREAD

**Discussion by WinFan3672** [i3/i3#5702] [Q&A] [answered]
Is it possible to have i3 running on one tty, say tty2, and then open another tty, say tty3, and have another i3 session?

**Comment by orestisfl** [comment]
Yes, any reason that prevents you from doing that?

**Comment by WinFan3672** [comment]
I use ly as my display manager, and I cannot launch multiple instances of it. I tried to launch it after running startx, but that creates 3 terminal windows that I cannot close. Launching from a tty gives me an error about not having the X server running.

**Reply by orestisfl** [reply, marked as answer]
To launch i3 from a tty you need `xinit`, see https://wiki.archlinux.org/title/xinit. TL;DR paste `exec i3` in ~/.xinitrc and then run `startx` from a tty.

**Reply by WinFan3672** [reply]
Thanks for clearing that up. That worked for me.
